org.lflang/src/org/lflang/validation/LFValidatorImpl.xtend

```diff
@@ -340,13 +340,13 @@ class LFValidatorImpl extends AbstractLFValidator {
 
     @Check(FAST)
     def checkWidthSpec(WidthSpec widthSpec) {
-        if (this.target != Target.C && this.target != Target.CPP && this.target != Target.Python) {
+        if (this.target != Target.C && this.target != Target.CCPP && this.target != Target.CPP && this.target != Target.Python) {
             error("Multiports and banks are currently only supported by the C and Cpp targets.",
                 Literals.WIDTH_SPEC__TERMS)
         } else {
             for (term : widthSpec.terms) {
                 if (term.parameter !== null) {
-                    if (this.target != Target.C && this.target != Target.Python && this.target != Target.CPP) {
+                    if (this.target != Target.C && this.target != Target.CCPP && this.target != Target.Python && this.target != Target.CPP) {
```

Comment:
Sidenote: I thought they were supported for `CPP`?

Reply:
They are supported in C, Python and C++, but not all the other targets. Notice the `!=` check. I think we could really do a better job in keeping track of the features supported by the targets. The code would be easier to read if each target would have a `supportsParameterixedWidths()` method.